packages-backend/express/src/middlewares/fixtures/jwt-token.ts

```diff
@@ -1,19 +1,49 @@
-import { JWT, JWK, JWKS } from 'jose';
+import {
+  exportJWK,
+  generateKeyPair,
+  type KeyLike,
+  SignJWT,
+  type JWK,
+} from 'jose';
 
-const keyRS256 = JWK.generateSync('RSA', 2048, { use: 'sig', alg: 'RS256' });
+let keyRS256: KeyLike;
+let jwksStore: { keys: JWK[] };
 
-const jwksStore = new JWKS.KeyStore([keyRS256]);
+async function initialize() {
```

Comment:
`jose` APIs are now async so I found this initializer the simplest way to go.

Reply:
You can also check how we initialize this in our MC services (`auth` package).

Anyway, if it does the job all good.